sdk/textanalytics/ai-text-analytics/src/generated/models/mappers.ts

```diff
@@ -472,34 +500,20 @@ export const Entity: coreHttp.CompositeMapper = {
       },
       category: {
         required: true,
-        serializedName: "type",
+        serializedName: "category",
```

Comment:
this change should also be added to the changelog

Reply:
This isn't an api-level change because we were already renaming this property in the convenience layer.